.gitlab-ci.yml

```diff
@@ -0,0 +1,128 @@
+---
+include:
+  - project: invenia/gitlab-ci-helper
+    file: /templates/julia.yml
+
+variables:
+  STACK_NAME_PREFIX: sandbox-awsbatch
+  CLOUDSPY_IMAGE: 468665244580.dkr.ecr.us-east-1.amazonaws.com/cloudspy
+
+# Variables that require shell execution or depend on a variable that does
+.setup: &setup
+  |
+  echo "$ci_init" > ci_init && source ci_init && rm ci_init
+  STACK_NAME=$(stack_name $STACK_NAME_PREFIX)
+  ACCOUNT_ID=$(aws_account_id)
+
+"Setup Environment":
+  stage: setup
+  except:
+    - tags
+    - master
+    - /^.+\/.*master$/  # e.g. jh/validate-master
+  when: always
+  environment:
+    name: branch/$CI_COMMIT_REF_SLUG
+    on_stop: "Delete Environment"
+  script:
+    - echo "Setting up environment"
+
+"Create Stack":
+  stage: setup
+  image: $CLOUDSPY_IMAGE
+  tags:
+    - docker-ci
+    - ci-account
+    # TODO: Needed until cloudspy supports multi-arch images:
+    # https://gitlab.invenia.ca/invenia/cloudspy/-/issues/85
```

Comment:
Not true anymore as this is open-source and doesn't use cloudspy?

Reply:
shouldn't this whole file be deleted since it's on github now?

Reply:
Yes. I'll make those changes.